vanilla-framework/vanilla-boilerplate-theme
# Add ./run scripts

## Done

Add ./run scripts for consistency

## QA

Ensure this theme is set up correctly by runnign through the various scripts.

Fixes #3
